### DISCUSSION TITLE
[Bug]: Academy - Certificate Pimcore Developer Senior - Caching Chapter

### DISCUSSION THREAD

**Discussion by Siqu** [pimcore/pimcore#15023] [General]
### Expected behavior

I am currently doing the Senior Course and cannot pass the Caching Chapter. By watching the video, the information about the output cache key is, that it contains the URL, WebP Support and if it is an ajax or a normal request.

When looking at the code inside of the FullPageCacheListener the key is generated by creating a md5 from the request uri, the method and from the pimcore_cache_tag_suffix variable.
The Screenresolution as desktop/mobile is used as an appendix after the md5 hash.

After trying multiple different answers i think that there is an error inside of the validation.

### Actual behavior

Question 5: What information is part of output cache key?

- [ ] URL
- [ ] Parameters
- [ ] Screen resolution
- [ ] WebP Support

### Steps to reproduce

Following answers are incorrect:
- [X] URL
- [X] Parameters
- [ ] Screen resolution
- [ ] WebP Support

OR

- [X] URL
- [ ] Parameters
- [ ] Screen resolution
- [X] WebP Support

OR

- [X] URL
- [X] Parameters
- [ ] Screen resolution
- [X] WebP Support

OR

- [X] URL
- [X] Parameters
- [X] Screen resolution
- [ ] WebP Support

OR

- [X] URL
- [ ] Parameters
- [X] Screen resolution
- [X] WebP Support

OR

- [X] URL
- [X] Parameters
- [X] Screen resolution
- [X] WebP Support

**Comment by kingjia90** [comment]
> * [x]  URL
> * [x]  Parameters
> * [ ]  Screen resolution
> * [x]  WebP Support

for me it returns as a correct answer



For more details, see also https://github.com/pimcore/pimcore/discussions/12074#discussioncomment-2717653

**Reply by Siqu** [reply]
Thats' strange, last week when i tried this combination it returned an error. 
Today this answer worked, thank you.
